### ISSUE TITLE
PHP 8 / PSR12: Named Arguments w/Nested Multi-line Arguments generate invalid indention errors

### ISSUE THREAD

**Issue by zimzat** [squizlabs/PHP_CodeSniffer#3253]
**Describe the bug**
Named Arguments with nested multi-line function calls generate invalid unexpected indention errors.

**Code sample**
```php
<?php

array_fill_keys(
    keys: range(
        1,
        12,
    ),
    value: true,
);
```

**To reproduce**
1. Create a file called `test.php` with the code sample above...
2. Run `phpcs -s test.php --standard=PSR12`
3. See error message displayed
```
FILE: test.php
----------------------------------------------------------------------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
----------------------------------------------------------------------------------------------------------------------------------
 5 | ERROR | [x] Multi-line function call not indented correctly; expected 4 spaces but found 8
   |       |     (PSR2.Methods.FunctionCallSignature.Indent)
 6 | ERROR | [x] Multi-line function call not indented correctly; expected 4 spaces but found 8
   |       |     (PSR2.Methods.FunctionCallSignature.Indent)
----------------------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------------------------------------------------------------------

Time: 35ms; Memory: 6MB
```

**Expected behavior**
The expected indention should match the declared indention, no error should be emitted.

**Versions:**
 - OS: Linux Ubuntu 18.04.1
 - PHP: 8.0.2
 - PHPCS: dev-master 1106d65547e581a5184a9293486d9e6d81595462
 - Standard: PSR12

**Additional context**
If you apply `phpcbf` to the code it will unindent lines 5 & 6, "fail to fix", and then `phpcs` will complain that it expects 8 spaces but found 4 on those same lines. ¯\\\_(ツ)\_/¯

My original code had multiple nested multi-line named arguments, but the above is the 'minimal' code necessary to generate the unexpected error.

**Comment by jrfnl** [comment]
@zimzat Thanks for reporting this! I've just pulled PR #3254 which should fix this. Would you mind testing the PR ?

**Comment by zimzat** [comment]
That's awesome, thanks! I've tested it and no longer get unexpected indention errors on any of the effected lines. Thanks for the quick turnaround! :tada:

**Comment by jrfnl** [comment]
@zimzat Thanks for testing and confirming the fix.

**Comment by gsherwood** [comment]
Note that I didn't classify this as a bug as PHPCS doesn't have support for name params at all. The changelog item covering this is the introduction of named params, but thank you for the report. If 3.4.0 was released without this change, it would very much be a bug report. Huge thanks to @jrfnl for actually fixing it.
